components/LoginButton/WarpcastButton.tsx

```
@@ -26,6 +26,11 @@ export function WarpcastButton({ className = "" }: WarpcastButtonProps) {
     connect({ connector: config.connectors[0] });
   };
 
+  const handleLogout = () => {
```

ESLint

[error] 'handleLogout' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)